src/sparseml/transformers/finetune/data/data_helpers.py

```diff
@@ -12,12 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Any, Dict, Optional
+from typing import Any, Callable, Dict, List, Optional
 
+import torch
 from datasets import Dataset, load_dataset
+from torch.utils.data import DataLoader, RandomSampler
+from transformers.data import default_data_collator
 
 
-__all__ = ["get_raw_dataset", "make_dataset_splits"]
+__all__ = ["format_calibration_data", "get_raw_dataset", "make_dataset_splits"]
+
+
+def format_calibration_data(
+    tokenized_dataset: Dataset,
+    num_calibration_samples: int,
```

Comment:
```suggestion
    num_calibration_samples: Optional[int],
```
num_calibration_steps can be None as well